frontend/src/components/Credentials/GCredentialDetailsItemContent.vue

```diff
@@ -84,8 +87,9 @@ export default {
       }
       return [
         {
-          label: 'Unknown',
-          value: 'Information not available',
+          label: this.credential?.kind || 'Unknown',
+          value: `No details available for credentials of type ${this.providerType}`,
```

Comment:
the following should be sufficient and also aligns with the other message that we display for shared credentials, which is "Details not available for shared credentials"
```suggestion
          value: 'Details not available',
```